packages/client/ui/react-ui/src/providers/CrossmintWalletProvider.tsx

```diff
@@ -54,31 +71,73 @@ export function CrossmintWalletProvider({
             return { startedCreation: false, reason: `Jwt not set in "CrossmintProvider".` };
         }
 
-        const internalCall = async () => {
+        if (enablePasskeyPrompt && config.signer && "type" in config.signer && config.signer.type === "PASSKEY") {
```

Comment:
bit of a loaded condition but ts wasn't aware of `condig.signer.type` existing before checking the signer